server/src/service/DfspNetworkConfigService.js

```diff
@@ -166,19 +179,138 @@ const createDFSPIp = async (ctx, dfspId, body, direction) => {
   return endpointItem;
 };
 
+/**
+ * Adds a new configuration to the DFSP Endpoint Egress
+ *
+ * @param {String} dfspId DFSP id
+ * @param body {Object} DFSPEgress
+ * @returns BaseEndpoint + DFSPEgress
+ **/
+ exports.createDFSPEgress = async (ctx, dfspId, body) => {
+  await PkiService.validateDfsp(ctx, dfspId);
+  const resultId = await DFSPEndpointModel.create(dfspId, StatusEnum.NOT_STARTED, null, DirectionEnum.EGRESS, body);
+  return transformEndpointModeltoApiRes(await DFSPEndpointModel.findObjectById(resultId));
+
+  // ## Example response
+  // return {
+  //   "id": 0,
+  //   "type": "IP",
+  //   "state": "NEW",
+  //   "createdDate": "2022-03-24T09:41:21.263Z",
+  //   "createdBy": "string",
+  //   "dfspId": "string",
+  //   "IPList": [
+  //     {
+  //       "description": "Notification Callback Egress IP & Ports",
+  //       "address": "163.10.24.28/30",
+  //       "ports": [
+  //         "80",
+  //         "8000-8080"
+  //       ]
+  //     }
+  //   ]
+  // };
```

Comment:
I think it worth adding that to the openAPI schema, rather than placing it here: 
https://swagger.io/docs/specification/adding-examples/
Also there is a plan to convert this project to TS that should help with identifying return types.